src/panels/lovelace/editor/unused-entities/hui-unused-entities.ts

```diff
@@ -86,54 +75,58 @@ export class HuiUnusedEntities extends LitElement {
             : ""}
         </div>
       </ha-card>
-      <search-input @value-changed=${this._handleSearchChange}></search-input>
       <ha-data-table
-        .headers=${[
-          {
-            name: "Entity",
+        .columns=${{
+          entity: {
+            title: "Entity",
             sortable: true,
-            direction: this._sortColumn === 0 ? this._sortDirection : null,
+            filterable: true,
+            filterKey: "friendly_name",
+            direction: "desc",
+            template: (stateObj) => html`
+              <state-badge
+                .hass=${this.hass!}
```

Comment:
So here it becomes interesting. By having this object be defined on every render, you are able to have `hass` state changes go through. Once this becomes a static variable, not so much.